SECURITY.md

```
@@ -184,7 +184,7 @@ Our security posture is continuously monitored and measured:
 - **Bandit**: Python security issue detection
 
 ### OpenSSF Scorecard
-- **Current Score**: View at https://github.com/VirtualAgentics/coderabbit-conflict-resolver/security
+- **Current Score**: View at https://github.com/VirtualAgentics/review-bot-automator/security
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)